src/ruby/ext/grpc/rb_channel_args.c

```diff
@@ -111,6 +111,8 @@ typedef struct channel_convert_params {
 static VALUE grpc_rb_hash_convert_to_channel_args0(VALUE as_value) {
   ID id_size = rb_intern("size");
   VALUE grpc_rb_cChannelArgs = rb_define_class("TmpChannelArgs", rb_cObject);
```

Comment:
By the way, is `TmpChannelArgs` being a top-level class being intented? It seems weird.